### PR TITLE
Add README with basic instruction

### PR DESCRIPTION
**Pull request by kordianbruck**
So people actually know how to use this amazing piece of :cake:

### CODE REVIEW

**Comment by kordianbruck** [comment]
@mwuertinger wupps, sorry, that slipped in there somehow. I've also added syntax highlighting for that extra fancyness

**Comment by kordianbruck** [comment]
@v-antech cause I usually force push to my branch for minor changes. You can check by the updated timestamp on the commit ;)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    }
+    defer influxClient.Close()
+
+    // Register our custom exporter to opencensus
+    view.RegisterExporter(influxdb.NewExporter(influxClient, influxDB, func(err error) {
+        log.Fatal(err)
+    }))
+
+    // Useful metrics to be exported from opencensus using the exporter
+	view.Register(ochttp.ServerRequestCountByMethod)
```
Comment (v-antech) [inline README.md]: Wasn't that the place in question - about using tabs?

Comment (kordianbruck) [reply README.md]: I finally found the 'show whitespace' option in vscode. I hope to fixed all of them now.

**Comment by v-antech** [comment]
Looks good - merging pull request - thanks :+1:

**Pull request merged**